iwyu_output.h

```diff
@@ -320,6 +368,10 @@ class IwyuFileInfo {
     return direct_includes_as_fileentries_;
   }
 
+  void RecordDeclInNamespaces(const clang::NamedDecl* decl);
```

Comment:
I understand that `RecordDecl` is a verb phrase here ("record the declaration"), but `RecordDecl` is also a type in Clang ("a record declaration"). Would be nice to find a different word than `Record` :-)